classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
```

Comment:
- [ ]  it would be better to add a `classroom` setter in the `Student` class as well in  order to achieve `belongs_to ` and make sure that method add the student to the classroom if it's not there already

Reply:
Resolved as requested